### ISSUE TITLE
Wrap long lines of text

### ISSUE THREAD

**Issue by vsdmub** [lallulli/songpress#31]
Hello. 

Thank you for you good program. It is almost perfect for me. Only it needs the wrap function for a long lines of text. By example:

[D]Cantad [A]a[D]le[G]gr[A]es [Bm]al [E7]Se[A]ñor, [D]mortales [G]to[D]dos [Em]por [Bm]do[A]quier, [F#]servidle [Bm]siem[D]pre [G]con fer[A]vor, [D]o[G]be[A]de[Bm]ced[G]le [D]con [A]pla[D]cer.

I can not see this song complete without splitting it. Would you like to implemment the wrap function for  long lines of text, please?

**Comment by lallulli** [comment]
Yes, we could introduce the concept of line length (with a sort of ruler to set it up), and use it to wrap long text lines; but we should think carefully about the right way to do it. For example:

- wrapped lines should be indented, so that they can be visualized as "continuation" of the previous line;
- even better, the user should be able to set "preferred" breaking points. In lyrics, the best place to break up a line is often in the middle of the line itself. In this case, indentation should not take place. But in this case, probably _all_ the lines should be broken in the middle. In other words, the song should support the concept of "large mode" and "narrow mode"; in narrow mode all the width of every line halves, while the number of lines doubles.

**Comment by vsdmub** [comment]
Ok. Thank you! I will waiting for the next release. Thank you for your work.
